### ISSUE TITLE
missing mysql/my_config.h

### ISSUE THREAD

**Issue by vitalije** [pcmacdon/jsish#38]
I have installed libmysqlclient-dev package on my Ubuntu 20.04. Yet, `make` reports an error:
```
src/jsiMySql.c:30:10: fatal error: mysql/my_config.h: No such file or directory
   30 | #include <mysql/my_config.h>
```
There is no `my_config.h` in `/usr/include/mysql/` folder. My version of `libmysqlclient-dev` is `8.0.19-0ubuntu5`

**Comment by pcmacdon** [comment]
Yes, it looks like mysql support is being strangled, so I've removed it from the build configs.
You may need to "rm make.conf" to see the change.
Going forward I think jsish will have to adopt mariadb.

**Comment by vitalije** [comment]
Thanks. It compiles now correctly and it solved also my other issue #37
Thanks
